.github/assets/template.md

```diff
@@ -0,0 +1,8 @@
+This is preserved as a backup for the yet to be implemented feature: Scheduled tweets. Do not delete this.
+
+:warning: Not implemented feature. Please leave as it is.
+
+**Provide a human friendly time for the tweet to go out**
+<!-- Input the scheduled time in conventional UTC format as follows: "2020-10-04T16:02:00.000Z" format. Please use `new Date().toISOString()` in JavaScript to get the date format -->
+
+>Time:
```

Comment:
Do we want to keep this file? I assume keeping a non implemented feature, although with a warning, might end up confusing people. Would removing this have any impact on the workflow?

Reply:
agreed! removed the file.